security/pkg/nodeagent/caclient/client.go

```diff
@@ -79,7 +79,7 @@ func getCATLSRootCertFromConfigMap(controller configMap, interval time.Duration,
 			break
 		}
 		time.Sleep(retryInterval)
-		log.Infof("unalbe to fetch CA TLS root cert: %v, retry in %v", err, interval)
+		log.Infof("Unable to fetch CA TLS root cert: %v, retry in %v", err, interval)
```

Comment:
This can be error level IMO

Reply:
Changed to the warning level. This doesn't seem like an error since there are retries.